.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - oraclejdk7
+  - oraclejdk8
 before_install:  
  - "echo $JAVA_OPTS"
  - "export JAVA_OPTS=-Xmx512m"
```

Comment:
@ajs6f I only changed the jdk specification (oraclejdk7 -> oraclejdk8) -- the heap size was unchanged.